docs/docs/applying.md

```diff
@@ -0,0 +1,117 @@
+# Applying JSpecify Nullness Annotations
+
+If your code has no nullness annotations, we recommend starting with one class
+or package at a time, ideally one that depends on as few others as possible.
+Then move outward from called code to its callers.
+
+In the future, we'll have suggestions for
+[tools that can automate some of this process](https://github.com/jspecify/jspecify/issues/553).
+
+## 1. Look for nullable type usages
+
+Look for methods, parameters, and fields where some part of the type component
```

Comment:
```suggestion
Look for return types, parameters, and fields where some part of the type component
```

The `methods` doesn't make clear what part to look at. Alternatively, remove the `parameters`, as that would be subsumed by `methods`.

Reply:
How's this?